src/oscar/apps/offer/abstract_models.py

```diff
@@ -551,6 +552,7 @@ class AbstractBenefit(BaseOfferMixin, models.Model):
     TYPE_CHOICES = (
         (PERCENTAGE, _("Discount is a percentage off of the product's value")),
         (FIXED, _("Discount is a fixed amount off of the basket's total")),
+        (FIXED_PRODUCT, _("Discount is a fixed amount off of the product's value")),
```

Comment:
Discount is a fixed amount off of every product's value in the entire basket.

Reply:
Don't listen to viggo, this is fine.